.changeset/nasty-crabs-worry.md

```diff
@@ -0,0 +1,25 @@
+---
+'astro': minor
+---
+
+Adds a new property to the globals `Astro` and `APIContext` called `routePattern`. The `routePattern` represents the current route (component)
+that is being rendered by Astro. It's usually a path pattern will look like this: `blog/[slug]`:
+
+```asto
+---
+// src/pages/blog/[slug].astro
+const route = Astro.routePattern;
+console.log(route); // it will log "blog/[slug]"
+---
+```
+
+```js
+// src/pages/index.js
+
+export const GET = (ctx) => {
+  console.log(ctx.routePattern) // it will log src/pages/index.js
```

Comment:
@ematipico I know this has already merged, but according to the line in docs about `index.*` showing only `/`, is this code comment about what's logged correct? It also includes the file extension here. 

We will proofread all the changesets before 5.0 beta release anyway, so I'm not concerned about updating this right now. but I just want to make sure I'm understanding correctly!